src/graphql/epoch.graphql

```diff
@@ -6,3 +6,32 @@ query Epoch {
     slot
   }
 }
+
+query InflationRate {
+  inflationRate: actions_inflation_rate {
+    validator
+    foundation
+    total
+    epoch
+  }
+}
+
+query InflationGovernor {
+  inflationGovernor: actions_inflation_governor {
+    initial
+    terminal
+    taper
+    foundation
+    foundationTerm: foundation_term
+  }
+}  
+
+query EpochSchedule {
+  epochSchedule: actions_epoch_schedule {
+    slotsPerEpoch: slots_per_epoch
+    leaderScheduleSlotOffset: leader_schedule_slot_offset
+    warmup
+    firstNormalEpoch: first_normal_epoch
+    firstNormalSlot: first_normal_slot
+  }
+}
```

Comment:
I would suggest combining this query in to one query called `EpochDetails`

Reply:
Updated.